source/_posts/2020-09-18-mac-companion.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "Home Assistant Companion for macOS"
+description: "Integrate your Mac into Home Assistant."
+date: 2020-09-16 02:00:00
+date_formatted: "September 18, 2020"
+comments: true
+author: Tom Brien
+author_twitter: tbrien88
+categories: Announcements
+og_image: /images/blog/2020-09-18-mac-companion/social.png
+---
+
+TODO: ADD VIDEO
+
+Wow what a birthday week it has been! We've had a [new supervisor release](/blog/2020/09/16/supervisor-joins-the-party), one of the largest and most user-driven core [releases](/blog/2020/09/17/release-115) thanks to the month of What The Heck?! and even seen the integration of [RFID tags](/blog/2020/09/15/home-assistant-tags) right into the heart of Home Assistant. To finish the week off we've got one last present for all our Mac users, we're announcing the beta release of Home Assistant Companion for macOS. Like a lot of the recent updates to the iOS app, we have [@zacwest](https://github.com/zacwest) to thank for this. Zac has ported the iOS app over to Mac and added some great new features specific to the Mac. If you're a Windows user, don't worry, you can integrate your PC with the great [IOT Link](https://iotlink.gitlab.io/index.html) tool.
+
+<p class='img'>
+<img src='/images/blog/2020-09-18-mac-companion/render.png' alt='Rendered image of the Home Assistant Lovelace interface running full screen on a 16-inch Mac Book Pro'></a>
+Rendered image of the Home Assistant Lovelace interface running full screen on a 16-inch Mac Book Pro.
+</p>
+
+## Automate when you start using your webcam or microphone
+
+Let's start by looking at some of the coolest Mac-specific features of the app, sensors for all your webcams and microphones. Each camera and microphone has its own `binary_sensor` showing whether it is active or not. These can enable some really useful automations, especially for those home working at the moment. You could automatically turn off the radio when answering a call or close the blinds behind you to improve your video quality. To see just how useful this can be in the real world, check out this video of how our very own Frenck is using these sensors in his streaming set up.
```

Comment:
```suggestion
There are several new `binary_sensor` available from your Mac: whether it's active, whether a particular microphone is in use, and whether a particular webcam is in use.

The "active" sensor reports whether the Mac is under active use; in other words, not sleeping, not showing a screensaver, not locked, and not idle. You can configure the Idle time threshold from its default 5 minutes in Preferences.

Each camera and microphone has its own `binary_sensor` showing whether it is active or not. These can enable some really useful automations, especially for those home working at the moment. You could automatically turn off the radio when answering a call or close the blinds behind you to improve your video quality. To see just how useful this can be in the real world, check out this video of how our very own Frenck is using these sensors in his streaming set up.
```